Assets/Prefabs/Input/Player.prefab

```diff
@@ -294,6 +294,7 @@ MonoBehaviour:
   - {fileID: 426369848560938732}
   animatorToDisable: {fileID: 3755527737191155748}
   rigidbodyToPush: {fileID: 4321660441894553452}
+  shadowDecal: {fileID: 5212658928600365944}
```

Comment:
hm, I see lots of changes in the derived prefabs, try applying as override to the player prefab in one after removing the change in the rest?

Reply:
The others are all separate prefabs, we actively use at least 5 different prefabs for the player whether it's AI or not, Tutorial or not, Networked or not, etc.

Reply:
Nah they aren't separate actually, they all inherit from Player :))))